### ISSUE TITLE
[data request] NIH Deep Lesion Dataset

### ISSUE THREAD

**Issue by jason-zl190** [tensorflow/datasets#1225]
* Name of dataset: NIH Deep Lesion Dataset
* URL of dataset: https://www.nih.gov/news-events/news-releases/nih-clinical-center-releases-dataset-32000-ct-images
* License of dataset: Unknown
* Short description of dataset and use case(s): The National Institutes of Health’s Clinical Center has made a large-scale dataset of CT images publicly available to help the scientific community improve detection accuracy of lesions. While most publicly available medical image datasets have less than a thousand lesions, this dataset, named DeepLesion, has over 32,000 annotated lesions identified on CT images. The images, which have been thoroughly anonymized, represent 4,400 unique patients, who are partners in research at the NIH.

Folks who would also like to see this dataset in `tensorflow/datasets`, please thumbs-up so the developers can know which requests to prioritize.

And if you'd like to contribute the dataset (thank you!), see our [guide to adding a dataset](https://github.com/tensorflow/datasets/blob/master/docs/add_dataset.md).

**Comment by jason-zl190** [comment]
#1224

**Comment by jason-zl190** [comment]
@Ouwen I closed the old issue because it's outdated. Would you mind be my reviewer? This version supports no long need to manually download and place the data.

**Comment by jason-zl190** [comment]
the link of the old issue: https://github.com/tensorflow/datasets/issues/853

**Comment by anigasan** [comment]
Hi, Can I work on this dataset?

**Comment by jason-zl190** [comment]
> Hi, Can I work on this dataset?

Of course, you can take a look at #1224 if you don't mind. I was busy on my final the past two weeks. I plan to have two updates next: beam download; different configs for different tasks — for example, classification and 3d volume for detection.

**Comment by jason-zl190** [comment]
the lastest PR  #1769
